css/build.css

```
@@ -201,3 +206,14 @@ input.form-control:focus {
 [data-widget-package="com.fliplet.login"] {
   width: 100%;
 }
+
+.fa-eye {
```

Please add more CSS context, otherwise this could affect any FA eye icon.

```suggestion
[data-widget-package="com.fliplet.login"] .fa-eye {
```